### ISSUE TITLE
Deal with transparency

### ISSUE THREAD

**Issue by lthms** [GreenPix/dilia#32]
In a chipset, one color is often used as the “transparency color” that is it is not supposed to be displayed in the map.

One way to improve the chipset upload and management is to deal with this transparency color. I suspect it is not an easy change as it also impacts the renderer.

**Comment by Nemikolh** [comment]
Thanks for opening this issue! When we discussed that point with @Azamung the idea was to convert this pixel into a totally transparent one.

**Comment by lthms** [comment]
Oh, that would work. But this means we modify the chipset before saving it? (It is not a problem, I just want to be sure I understood correctly)

**Comment by Nemikolh** [comment]
Yes! That's idea. We were thinking of either detecting that automatically or provide a pen that would allow to select the color.

**Comment by lthms** [comment]
Probably we can imagine the following:
1. A way to set by hand a hexa color code (an input)
2. A pen to set this input to a selected pixel value
3. And maybe later a way to detect it automatically (but it should be afterwards) and set the value of the input with the result

This is three steps we can do one by one. What do you think?

**Comment by Nemikolh** [comment]
I like this incremental approach. I think we should implement it that way.
